### PR TITLE
Mkieburtz/circleci refactor

### PR DESCRIPTION
**Pull request by EtherealMC-Bit**
Created a bash script that contains the commands for `export`ing the environment variables used for building.

### CODE REVIEW

File: export_vars.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export WarpX_DIMS=2
+export WarpX_MPI=ON
+export WarpX_OPENPMD=OFF
+export WarpX_PSATD=OFF
+export WarpX_QED=OFF
+export WarpX_EB=ON
+export ME_Solver=ON
```
Comment (roelof-groenewald) [inline export_vars.sh]: This has now been removed in `memaster`.
```suggestion
```

**Review by roelof-groenewald — CHANGES_REQUESTED** [review]
The ME_Solver flag should be removed. Is it possible to move the `export_vars.sh` file to the `.circleci` folder? It just seems cleaner to put it there.

**Review by roelof-groenewald — APPROVED** [review]
Actually I'll just go ahead and approve this so you guys don't need to wait for me before merging this (but please do address the comments I left before merging).

**Review by peterscherpelz — CHANGES_REQUESTED** [review]
I'll mark this as request changes agreeing with Roelof's comments.

**Review by peterscherpelz — APPROVED** [review]

**Comment by peterscherpelz** [comment]
Thanks!!!

**Pull request merged**